assets/global.js

```diff
@@ -615,6 +616,12 @@ class VariantSelects extends HTMLElement {
     window.history.replaceState({ }, '', `${this.dataset.url}?variant=${this.currentVariant.id}`);
   }
 
+  updateShareUrl() {
+    if (!this.currentVariant) return;
+    console.log(document.querySelector("[data-share]").dataset, document.querySelector("[data-share]"))
+    document.querySelector("[data-share]").value = `${ document.querySelector("[data-share]").dataset.url}${this.dataset.url}?variant=${this.currentVariant.id}`;
```

Comment:
If there are multiple featured product sections on a page, `document.querySelector("[data-share]")` will return the first node that matches the selector, which is not necessarily the correct one

Reply:
Oh yes, good catch! I saw that it only allows for one `share` block, but I forgot about this scenario with the featured product. I will add a section id